src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/LogRecordExtensions.cs

```diff
@@ -75,8 +75,8 @@ internal static OtlpLogs.LogRecord ToOtlpLog(this LogRecord logRecord, SdkLimitO
                 otlpLogRecord.SeverityText = logRecord.Severity.Value.ToShortName();
             }
 
-            var attributeValueLengthLimit = sdkLimitOptions.AttributeValueLengthLimit;
-            var attributeCountLimit = sdkLimitOptions.AttributeCountLimit ?? int.MaxValue;
+            var attributeValueLengthLimit = sdkLimitOptions.LogRecordAttributeValueLengthLimit;
```

Comment:
isn't this a breaking change?

Reply:
No, because the property implementation ends up returning `AttributeValueLengthLimit` if the length limit was not set. Same for the attribute count limit on the next line.

Reply:
My bad, I missed the change on `SdkLimitionOptions`